packages/terra-form-input/src/terra-dev-site/doc/example/controlled/DefaultIncompleteExample.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Field from 'terra-form-field';
+import Input from 'terra-form-input';
+
+class DefaultIncompleteExample extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { input: 'Default Incomplete Input' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ input: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+        <Field label="Default" htmlFor="default-controlled-incomplete">
+          <Input
+            name="default incomplete input"
+            id="default-controlled-incomplete"
+            value={this.state.input}
+            onChange={this.handleChange}
+            required
+            isIncomplete
```

Comment:
The isIncomplete never changes in this example. I think we should either update the example to switch out of an incomplete state under some condition or squash this into a single isIncomplete example (I'm not sure how much value there is to having both a controlled/uncontrolled isIncomplete example).

Reply:
Changed it over here https://github.com/cerner/terra-core/pull/2670/commits/418492f37795d6dfbe0171c38fef54c1f5fd7577#diff-380a6189c6663fbefbe5cfb308f70390R33

I'm not sure how much value there is either. I just copied what we were doing for `isInvalid` examples here.